src/components/CreateResetPassword/components/PasswordRequirementsField/PasswordRequirementsField.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Layout } from '@folio/stripes-components';
+
+import styles from './PasswordRequirementsField.css';
+
+const passwordRequirementsNames = [
+  'password_length',
+  'numeric_symbol',
+  'special_character',
+  'alphabetical_letters',
+];
+
+class PasswordRequirementsField extends React.Component {
```

Comment:
Can we not use `PasswordValidationField` from `stripes-core` here? It seems that it does exactly what is needed in the story

Reply:
No, unfortunately it cannot be used here for a number of reasons.